tests/gnuhealth/gnuhealth_client_first_time.pm

```diff
@@ -10,17 +10,25 @@ use base 'x11test';
 use strict;
 use warnings;
 use testapi;
-use version_utils 'is_leap';
+use version_utils qw(is_leap is_tumbleweed);
 
 sub run {
     my $gnuhealth = get_var('GNUHEALTH_CLIENT', 'gnuhealth-client');
     my $gnuhealth_34 = is_leap('<15.2');
+    my $gnuhealth_4 = is_tumbleweed() || is_leap('>=15.4');
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version – you should like that ;). That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
I was/am aware of that.. with gnuhealth being tracked only in Leap and TW, the alternative was

```
if (!is_leap('<15.4')) {

}
```

which looks rather silly to me - anyway: PR has been updated now